src/common/post.ts

```diff
@@ -126,33 +126,37 @@ export const createSquadWelcomePost = async (
   args: Partial<FreeformPost> = {},
 ) => {
   const content = getWelcomeContent(source);
-  const repo = con.getRepository(WelcomePost);
   const id = await generateShortId();
 
-  return repo.save({
-    ...args,
-    id,
-    shortId: id,
-    title: WELCOME_POST_TITLE,
-    sourceId: source.id,
-    authorId: adminId,
-    content,
-    contentHtml: markdown.render(content),
-    image: defaultImage.welcomePost,
-    banned: true,
-    flags: {
+  return savePost({
+    con,
+    type: WelcomePost,
+    shouldUpdateFlags: true,
+    post: {
```

Comment:
Nothing was changed with the object we passed as `post`, just indentation due to being inside an object.